index.js

```diff
@@ -219,11 +219,11 @@ const main = async () => {
       },
       repositoryName: repository
     }).promise()
-    status = 'IN_PROGRESS'
+    status = 'PENDING'
```

Comment:
Is this change necessary?

Reply:
I think the flow will be `PENDING` -> `IN_PROGRESS` -> `ACTIVE` / `COMPLETE`, but I'm not sure. 🙈 

The behavior should be the same either way, so I will revert the change if you want. 👍